roles/amq_broker/defaults/main.yml

```diff
@@ -28,7 +28,12 @@ activemq_jolokia_url: "http://{{ activemq_host }}:{{ activemq_http_port }}/conso
 activemq_console_url: "http://{{ activemq_host }}:{{ activemq_http_port }}/console/"
 activemq_jvm_package: java-11-openjdk-headless
 activemq_java_home:
-activemq_java_opts: "-Xms512M -Xmx2G -XX:+PrintClassHistogram -XX:+UseG1GC -XX:+UseStringDeduplication -Dhawtio.disableProxy=true -Dhawtio.realm=activemq -Dhawtio.offline=true -Dhawtio.rolePrincipalClasses=org.apache.activemq.artemis.spi.core.security.jaas.RolePrincipal -Djolokia.policyLocation=${ARTEMIS_INSTANCE_ETC_URI}jolokia-access.xml "
+activemq_java_opts: "{{ [
```

Comment:
@guidograzioli be careful, that's getting to complicated for default. Down the road, it might be better to assemble that (somehow) dynamically to be able to check all vars involved and ensure that the array does not blow up to the user.

Reply:
agreed... worth mentioning that's the current default value in amq